draft-ietf-teep-architecture.md

```diff
@@ -550,7 +553,7 @@ the Trusted Component back to the Trusted Component Signer. The Trusted Componen
 some cases might be another party such as a Device Administrator
 or other party
 to whom the code has been licensed (in which case the same code might
-be signed by multiple licensees and distributed as if it were different TAs).
+be signed by multiple licensees and distributed as if it was different TAs).
```

Comment:
```suggestion
be signed by multiple licensees and distributed as if it were different TAs).
```
original grammar is actually correct.  see https://www.grammarly.com/blog/conditional-sentences-was-instead-of-were/